### DISCUSSION TITLE
Repository name: Open-Shell-Menu -> Open-Shell?

### DISCUSSION THREAD

**Discussion by bonzibudd** [Open-Shell/Open-Shell-Menu#1258] [General]
It would make more sense, since all utilities are included in this repository. I know that all links should eventually be updated to match, but I believe the old links will still work.

**Comment by ge0rdi** [comment]
I think there was some plan to split the components (start menu, explorer extension) into separate repos eventually,
But I'm not sure how actual (and reasonable) that plan is.

I think it would make more sense to have repo named `Open-Shell`.
Most of links should work as before. 
Though it seems that URL of official page (`https://open-shell.github.io/Open-Shell-Menu`) will change 😞 

https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

> If you plan to rename a repository that has a GitHub Pages site, we recommend using a custom domain for your site. This ensures that the site's URL isn't impacted by renaming the repository.

Site links in old product versions will probably not work anymore.
And if anyone links to our page it will stop working as well.

I guess the `open-shell.github.io` page is quite used because it is second domain (right after google) in stats of pages that referred this repo.
So I guess we don't want to change the URL just like that ...
